Userland/Utilities/seq.cpp

```diff
@@ -85,14 +88,14 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
         return 1;
     }
 
-    if (__builtin_isnan(start) || __builtin_isnan(step) || __builtin_isnan(end)) {
+    if (__builtin_isnan(start) != 0 || __builtin_isnan(step) != 0 || __builtin_isnan(end) != 0) {
```

Comment:
If this change has an effect, then it definitely needs a comment

Reply:
It has no effect. It just resolves a clang-tidy warning about implicitly converting ints to bools.

Reply:
In general we convert ints, pointers, and whatever implicitly to bools. or at least we used to. I'll leave this one up to maintainers who are more up to date.

Reply:
All the existing users of `__builtin_isnan()` or `isnan()` that I can find just implicitly cast it to bool. I don't have strong feelings about it though.

Reply:
I've reverted this change and kept the conversion implicit.